build.gradle

```diff
@@ -12,20 +12,31 @@
  */
 
 
+import com.github.jk1.license.filter.LicenseBundleNormalizer
 import groovy.transform.Memoized
 import net.ltgt.gradle.errorprone.CheckSeverity
+import tech.pegasys.internal.license.reporter.GroupedLicenseHtmlRenderer
 
 import java.text.SimpleDateFormat
 
+buildscript {
+  repositories {
+    maven { url "https://artifacts.consensys.net/public/maven/maven/" }
+  }
+  dependencies {
+    classpath 'tech.pegasys.internal.license.reporter:license-reporter:develop'
```

Comment:
Is there a release version for the license reporter we can use?

Reply:
good catch, 1.0.0!